### PR TITLE
Add 1.20.0 changelog (#25901)

### PR DESCRIPTION
**Pull request by delvh**
Backport of #25901

### CODE REVIEW

**Review by yardenshoham — APPROVED** [review]

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,6 +4,621 @@ This changelog goes through all the changes that have been made in each release
 without substantial changes to our git log; to see the highlights of what has
 been added to each release, please refer to the [blog](https://blog.gitea.io).
```
Comment (JakobDev) [inline CHANGELOG.md:5]:
```suggestion
been added to each release, please refer to the [blog](https://blog.gitea.com).
```

Comment (delvh) [reply CHANGELOG.md:5]: Hmm… I mean, yes, we should fix it, but even on the release/v1.20 branch?
If at all, it's probably better to backport a separate PR for that.

Comment (delvh) [reply CHANGELOG.md:5]: Feared so:
```go
delvh@pc:~/git/gitea$ ag blog.gitea.io
docs/config.yaml
56:      url: https://blog.gitea.io/
108:          url: https://blog.gitea.io/
155:          url: https://blog.gitea.io/
202:          url: https://blog.gitea.io/
249:          url: https://blog.gitea.io/
296:          url: https://blog.gitea.io/
343:          url: https://blog.gitea.io/

docs/content/doc/installation/upgrade-from-gitea.en-us.md
31:Before upgrade, please read the [Changelog on Gitea blog](https://blog.gitea.io/)

docs/content/doc/installation/upgrade-from-gitea.zh-cn.md
29:在更新前，请 [在 Gitea 博客上阅读更新日志](https://blog.gitea.io/)

docs/content/doc/usage/actions/comparison.en-us.md
34:See [Creating Go Actions](https://blog.gitea.io/2023/04/creating-go-actions/).

docs/content/doc/usage/actions/comparison.zh-cn.md
34:请参阅[创建Go Actions](https://blog.gitea.io/2023/04/creating-go-actions/)。
docs/content/page/index.en-us.md
14:Gitea was forked from [Gogs](http://gogs.io) originally and almost all the code has been changed. See the [Gitea Announcement](https://blog.gitea.io/2016/12/welcome-to-gitea/)

docs/content/page/index.fr-fr.md
13:Gitea est un service Git auto-hébergé très simple à installer et à utiliser. Il est similaire à GitHub, Bitbucket ou Gitlab. Le développement initial provient sur [Gogs] (http://gogs.io), mais nous l'avons forké puis renommé Gitea. Si vous souhaitez en savoir plus sur les raisons pour lesquelles nous avons fait cela, lisez [cette publication] (https://blog.gitea.io/2016/12/welcome-to-gitea/) sur le blog.

docs/content/page/index.zh-cn.md
13:Gitea 是一个自己托管的Git服务程序。他和GitHub, Bitbucket or Gitlab等比较类似。他是从 [Gogs](http://gogs.io) 发展而来，不过我们已经Fork并且命名为Gitea。对于我们Fork的原因可以看 [这里](https://blog.gitea.io/2016/12/welcome-to-gitea/)。
docs/content/page/index.zh-tw.md
14:Gitea 是從 [Gogs](http://gogs.io) Fork 出來的，請閱讀部落格文章 [Gitea 公告](https://blog.gitea.io/2016/12/welcome-to-gitea/)以了解我們 Fork 的理由。
CHANGELOG.md
5:been added to each release, please refer to the [blog](https://blog.gitea.io).

options/locale/locale_lv-LV.ini
3041:dashboard.new_version_hint=Ir pieejama Gitea versija %s, pašreizējā versija %s. Papildus informācija par jauno versiju ir pieejama <a target="_blank" rel="noreferrer" href="https://blog.gitea.io">mājas lapā</a>.

options/locale/locale_fr-FR.ini
2478:dashboard.new_version_hint=Gitea %s est maintenant disponible, vous utilisez %s. Consultez <a target="_blank" rel="noreferrer" href="https://blog.gitea.io">le blog</a> pour plus de détails.

options/locale/locale_el-GR.ini
2533:dashboard.new_version_hint=Το Gitea %s είναι διαθέσιμο, τώρα εκτελείτε το %s. Ανατρέξτε <a target="_blank" rel="noreferrer" href="https://blog.gitea.io">στο blog</a> για περισσότερες λεπτομέρειες.

options/locale/locale_ja-JP.ini
2533:dashboard.new_version_hint=Gitea %s が入手可能になりました。 現在実行しているのは %s です。 詳細は <a target="_blank" rel="noreferrer" href="https://blog.gitea.io">ブログ</a> を確認してください。
options/locale/locale_zh-TW.ini
2442:dashboard.new_version_hint=現已推出 Gitea %s，您正在執行 %s。詳情請參閱<a target="_blank" rel="noreferrer" href="https://blog.gitea.io">部落格</a>的說明。
options/locale/locale_tr-TR.ini
2465:dashboard.new_version_hint=Gitea %s şimdi hazır, %s çalıştırıyorsunuz. Ayrıntılar için <a target="_blank" rel="noreferrer" href="https://blog.gitea.io">blog</a>'a bakabilirsiniz.

options/locale/locale_pt-PT.ini
2533:dashboard.new_version_hint=O Gitea %s está disponível, você está a correr a versão %s. Verifique o <a target="_blank" rel="noreferrer" href="https://blog.gitea.io">blog</a> para mais detalhes.

options/locale/locale_en-US.ini
2634:dashboard.new_version_hint = Gitea %s is now available, you are running %s. Check <a target="_blank" rel="noreferrer" href="https://blog.gitea.io">the blog</a> for more details.

options/locale/locale_zh-CN.ini
2468:dashboard.new_version_hint=Gitea %s 现已可用，您正在运行 %s。查看 <a target="_blank" rel="noreferrer" href="https://blog.gitea.io">博客</a> 了解详情。
options/locale/locale_de-DE.ini
3037:dashboard.new_version_hint=Gitea %s ist jetzt verfügbar, Ihre derzeitige Version ist %s. Weitere Details finden Sie im <a target="_blank" rel="noreferrer" href="https://blog.gitea.io">Blog</a>.

options/locale/locale_ru-RU.ini
2522:dashboard.new_version_hint=Доступна новая версия Gitea %s, вы используете %s. Более подробную информацию читайте в <a target="_blank" rel="noreferrer" href="https://blog.gitea.io">блоге</a>.

README.md
67:[forked](https://blog.gitea.io/2016/12/welcome-to-gitea/) from

CONTRIBUTING.md
476:https://blog.gitea.io/2023/02/gitea-quarterly-report-23q1/
483:- 2023-01-01 ~ 2023-12-31 - https://blog.gitea.io/2023/02/gitea-quarterly-report-23q1/
```

**Review by denyskon — APPROVED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (denyskon) [reply CHANGELOG.md:5]: I think we can do a separate PR where we update the old URLs

**Pull request merged**